packages/react-jsx-highstock/test/test-helper.js

```diff
@@ -1,32 +1,8 @@
-import 'jest-enzyme';
-
-function copyProps(src, target) {
-  const props = Object.getOwnPropertyNames(src)
-    .filter(prop => typeof target[prop] === 'undefined')
-    .map(prop => Object.getOwnPropertyDescriptor(src, prop));
-  Object.defineProperties(target, props);
-}
-
-//const jsdom = new JSDOM('<!doctype html><html><body></body></html>');
-//const { window } = jsdom;
-window.Date = global.Date;
-
 const nodeCrypto = require('crypto');
 window.crypto = {
```

Comment:
I know I added this, but I have no idea why...

Reply:
I remember that I tried to remove it, but something broke..